opencog/atoms/execution/ExecutionOutputLink.cc

```diff
@@ -105,14 +101,19 @@ ExecutionOutputLink::ExecutionOutputLink(const Link& l)
 ///
 Handle ExecutionOutputLink::execute(AtomSpace* as, bool silent) const
 {
+	if (_outgoing[0]->getType() != GROUNDED_SCHEMA_NODE) {
+		LAZY_LOG_FINE << "Not a grounded schema. Do not execute it";
```

Comment:
?? I'm pretty sure you need to execute the lambda's and the defined schemas too.  Right?  Why wouldn't you want to do this?